modular_pariah/modules/radiosound/code/headset.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/radio/headset
+	var/radiosound = 'modular_pariah/modules/radiosound/sound/radio/common.ogg'
+
+/obj/item/radio/headset/syndicate //disguised to look like a normal headset for stealth ops
+	radiosound = 'modular_pariah/modules/radiosound/sound/radio/syndie.ogg'
+
+/obj/item/radio/headset/headset_sec
+	radiosound = 'modular_pariah/modules/radiosound/sound/radio/security.ogg'
+
+/obj/item/radio/headset/talk_into(mob/living/M, message, channel, list/spans, datum/language/language, list/message_mods, direct = TRUE)
+	if(radiosound && listening)
+		playsound(M, radiosound, rand(20, 30), 0, 0, SOUND_FALLOFF_EXPONENT)
```

Comment:
The last three arguments here (0, 0, SOUND_FALLOFF_EXPONENT) are already set to this by default, so they don't need to be specified:
https://github.com/pariahstation/Pariah-Station/blob/036a4b761f560d94686e02327c2758c78d4519dd/code/game/sound.dm#L44
```suggestion
		playsound(M, radiosound, rand(20, 30))
```